Adds EIS info to Security billing (#1070)
elastic/docs-content: deploy-manage/cloud-organization/billing/security-billing-dimensions.md

Logs, events, alerts, and configuration data ingested into your security project are billed using the **Ingest** and **Retention** pricing described above.

For more details about {{elastic-sec}} serverless project rates and billable assets, refer to Cloud Protection in the [{{ecloud}} pricing table](https://cloud.elastic.co/cloud-pricing-table?productType=serverless&project=security).

> ## Elastic Inference Service

Reviewer: ```suggestion
## Default LLM
```

> Elastic Inference Service (EIS) enables you to leverage AI-powered search as a service without deploying a model in your serverless project. EIS is configured as a default LLM for use with the Security AI Assistant, Attack Discovery, and other applicable AI features as a part of your "Complete" tier subscription. Using the default Elastic LLM will utilize EIS tokens and incur related token-based add-on billing for your serverless project.

Reviewer: ```suggestion
The default LLM enables you to leverage AI-powered search as a service without deploying a model in your serverless project. It is configured by default for use with the Security AI Assistant, Attack Discovery, and other applicable AI features as a part of your "Complete" tier subscription. Using the default LLM will use tokens and incur related token-based add-on billing for your serverless project.
```

Logs, events, alerts, and configuration data ingested into your security project are billed using the **Ingest** and **Retention** pricing described above.

For more details about {{elastic-sec}} serverless project rates and billable assets, refer to Cloud Protection in the [{{ecloud}} pricing table](https://cloud.elastic.co/cloud-pricing-table?productType=serverless&project=security).

% Note: Commenting out this section until this is ready to be published. 

> % ## Elastic-managed LLM

Reviewer 2: @jmikell821, the PLI, or what is shown on the billing usage page is still called 'Elastic Inference Service'. Only the dropdown for Elastic LLM when selecting the LLM in AI Assistant or Attack Discovery is now called Elastic Managed LLM (or Elastic-managed LLM, until James or Shubha confirm). So I recommend that the title of this section be called something like 'Elastic Inference Service - Elastic Managed LLM'. In the body we could replace "default LLM" with "default Elastic Managed LLM."

Author: Thanks @gaurav-elastic. Since I've merged the PR, I've added a note to https://github.com/elastic/docs-content/issues/967 as a reminder for when we're ready to publish this.
